### PR TITLE
fix(components): defaultHeight for VirtualizedList

### PR DESCRIPTION
**Pull request by frassinier**
**What is the problem this PR is trying to solve?**
VirtualizedList leads on its parent element for defining its height.
If its parent has height set to 0px (ie. inline element), it can not render any list item.
ngReact wrap React pure components into Angular component (which they are inline) and unit tests fail.

**What is the chosen solution to this problem?**
Introduce `defaultHeight` for VirtualizedList in order to not render it at 0px

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/VirtualizedList/VirtualizedList.component.js [hunk 1]
```diff
@@ -73,6 +74,7 @@ function VirtualizedList(props) {
 VirtualizedList.displayName = 'VirtualizedList';
 VirtualizedList.propTypes = propTypes;
 VirtualizedList.defaultProps = {
+	defaultHeight: 250,
```
Comment (jmfrancois) [inline packages/components/src/VirtualizedList/VirtualizedList.component.js]: question: why 250 ?

File: packages/components/src/VirtualizedList/VirtualizedList.test.js [hunk 2]
```diff
@@ -43,6 +43,34 @@ describe('VirtualizedList', () => {
...
+			</VirtualizedList>,
+		);
+
+		const renderer = wrapper.node.props.children;
+
+		// when
+		const rendererInstance = renderer({ height: 0, width: 300 });
+
+		// then
+		expect(rendererInstance).toMatchSnapshot();
```
Comment (jmfrancois) [inline packages/components/src/VirtualizedList/VirtualizedList.test.js]: I don't understand this tests. sorry. What do you really expect ?

Comment (frassinier) [reply packages/components/src/VirtualizedList/VirtualizedList.test.js]: I'm expecting `height: <DEFAULT_HEIGHT>` if AutoSizer returns `height: 0`

File: packages/components/src/VirtualizedList/VirtualizedList.component.js (hunk 1 above)
Comment (frassinier) [reply packages/components/src/VirtualizedList/VirtualizedList.component.js]: `$> ./randomValue.sh`, did you prefer another one?

Comment (frassinier) [reply packages/components/src/VirtualizedList/VirtualizedList.component.js]: I don't care, I don't want `height: 0` :)

File: packages/components/src/VirtualizedList/VirtualizedList.test.js (hunk 2 above)
Comment (jmfrancois) [reply packages/components/src/VirtualizedList/VirtualizedList.test.js]: In that case please remove snapshots and do
```javascript
expect(rendererInstance.props().height).toBe(600)
```

**Pull request merged**